.github/workflows/docs.yml

```diff
@@ -163,6 +163,23 @@ jobs:
         with:
           name: C-docs
           path: ./bindings/c/docs/doxygen/html
+  build-lua-doc:
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
Hi, you need to run `actions/checkout@v3` first.

Reply:
Yeap, I forget it. And one question, How can I get URL of my PR build doc?

Reply:
And I don't know why this PR CI failed in macos build. 😆

Reply:
Hi, sorry for that, we don't have a online preview now. But you could try to serve the docs built locally and ensure it accessable. If it works, it will works on our site, too.
And you can update here to add a link for it.
https://github.com/apache/incubator-opendal/blob/main/website/docusaurus.config.js#L129

Reply:
I've re-run the failed macos job for you. It's ok.